### PR TITLE
Add events to drawing manager.

### PR DESCRIPTION
**Pull request by jribble**

### CODE REVIEW

File: src/coffee/directives/api/models/parent/drawing-manager-parent-model.coffee [hunk 1]
```diff
@@ -3,6 +3,24 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
     ['uiGmapLogger', '$timeout'
       ($log, $timeout) ->
         class DrawingManagerParentModel
+          setEvents: (gObject, scope, model, ignores) ->
+            if angular.isDefined(scope.events) and scope.events? and angular.isObject(scope.events)
```
Comment (nmccready) [inline src/coffee/directives/api/models/parent/drawing-manager-parent-model.coffee]: why did you cut and paste this code, pull in the EventsHelper as a service / factory and use it directly. That way I manage the code in one spot.

Same goes for removeEvents.

**Comment by jribble** [comment]
I originally pulled in EventsHelper and included it.  Then I realized that EventsHelper sent in a model as one of the arguments to the event handler, which didn't make sense in this case.  But I'll change it back.  Easy enough.

**Comment by nmccready** [comment]
Says I can't merge please rebase or merge updates into yours.

**Comment by nmccready** [comment]
if you don't want model in Events Helper you could refactor it to handle two different callback styles w/ and without model.

**Comment by nmccready** [comment]
please update

**Comment by jribble** [comment]
It looks like this capability was already added last month and I wasn't up to date.  Sorry about that.

**Pull request closed without merging**